modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js

```diff
@@ -106,6 +108,80 @@ function BulkActions({
 		}
 	}
 
+	const renderNewBulkActions = ({
```

Comment:
A more React-y way to do this is to make a component instead of a funcion, something like `<BulkActionControls>`. Is there a reason gor `new` in name? :thinking: If yes, then maybe `<NewBulkActions>`

Reply:
This was supposed to be a temporary thing to put this behind the feature flag without making the indentations in the `render` too crazily indented.